src/UKHO.S100PermitService.Common/Extensions/ListExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.S100PermitService.Common.Extensions
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
Should really write a quick set of unit tests for this

Reply:
Added unit tests for ListExtensions class